new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -816,96 +1147,177 @@ private static String createAndVerifyDomainImage() {
+            + MONEXP_WDT_FILE);
+
+    wdtImage =
+        createImageAndVerify(MONEXP_IMAGE_NAME,
+                modelList,
+                appList,
+                propertyList,
+                WLS_BASE_IMAGE_NAME,
+                WLS_BASE_IMAGE_TAG,
+                WLS,
+                false,
+                domain2Uid, true);
+
+
+
+    // docker login and push image to docker registry if necessary
+    dockerLoginAndPushImageToRegistry(wdtImage);
+
+    // create docker registry secret to pull the image from registry
+    logger.info("Create docker registry secret in namespace {0}", domain2Namespace);
+    assertDoesNotThrow(() -> createDockerRegistrySecret(domain2Namespace),
+        String.format("create Docker Registry Secret failed for %s", REPO_SECRET_NAME));
+
+    return wdtImage;
+  }
+
+  //create domain from provided image and verify it's start
+  private static void createAndVerifyDomain(String miiImage,
+                                            String domainUid,
+                                            String namespace,
+                                            String domainHomeSource,
+                                            int replicaCount) {
     // create secret for admin credentials
     logger.info("Create secret for admin credentials");
     String adminSecretName = "weblogic-credentials";
-    assertDoesNotThrow(() -> createSecretWithUsernamePassword(adminSecretName, domain1Namespace,
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(adminSecretName, namespace,
         "weblogic", "welcome1"),
         String.format("create secret for admin credentials failed for %s", adminSecretName));
 
     // create encryption secret
     logger.info("Create encryption secret");
     String encryptionSecretName = "encryptionsecret";
-    assertDoesNotThrow(() -> createSecretWithUsernamePassword(encryptionSecretName, domain1Namespace,
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(encryptionSecretName, namespace,
         "weblogicenc", "weblogicenc"),
         String.format("create encryption secret failed for %s", encryptionSecretName));
 
     // create domain and verify
     logger.info("Create model in image domain {0} in namespace {1} using docker image {2}",
-        domainUid, domain1Namespace, miiImage);
-    createDomainCrAndVerify(adminSecretName, REPO_SECRET_NAME, encryptionSecretName, miiImage,domainUid);
-
+        domainUid, namespace, miiImage);
+    createDomainCrAndVerify(adminSecretName, REPO_SECRET_NAME, encryptionSecretName, miiImage,domainUid,
+            namespace, domainHomeSource, replicaCount);
+    String adminServerPodName = domainUid + "-admin-server";
     // check that admin server pod exists in the domain namespace
     logger.info("Checking that admin server pod {0} exists in namespace {1}",
-        adminServerPodName, domain1Namespace);
-    checkPodExists(adminServerPodName, domainUid, domain1Namespace);
+        adminServerPodName, namespace);
```

Comment:
you don't need this check, podReady does both

Reply:
done